lms/djangoapps/instructor/tests/test_enrollment.py

```diff
@@ -567,6 +575,45 @@ def test_bad_action(self):
 
 
 @attr('shard_1')
+class TestGetEmailParamsCCX(ModuleStoreTestCase):
+    """
+    Test what URLs the function get_email_params for CCX student enrollment.
+    """
+    @patch.dict('django.conf.settings.FEATURES', {'CUSTOM_COURSES_EDX': True})
+    def setUp(self):
+        super(TestGetEmailParamsCCX, self).setUp()
+
+        self.course = CourseFactory.create(default_store=ModuleStoreEnum.Type.split)
```

Comment:
Rather than using `default_store`, just set the `MODULESTORE` class attribute (as documented in the `ModuleStoreTestCase` docstring).